priv/repo/migrations/20170711183723_add_admin_guest_active_fields_to_users.exs

```diff
@@ -0,0 +1,11 @@
+defmodule Agoneum.Repo.Migrations.AddAdminGuestActiveFieldsToUsers do
+  use Ecto.Migration
+
+  def change do
+    alter table(:account_users) do
+      add :active, :boolean, default: false
+      add :guest, :boolean, default: false
+      add :admin, :boolean, default: false
```

Comment:
(same comment as [this](https://github.com/ElixirCLE/agoneum/pull/6/files#r126796167) )